onnxutils/onnxtk/metadata_props.py

```diff
@@ -0,0 +1,83 @@
+import warnings
+from .common.case_insensitive_dict import CaseInsensitiveDict
+import onnx
+from onnx import onnx_pb as onnx_proto
+
+KNOWN_METADATA_PROPS = CaseInsensitiveDict({
+    'Image.BitmapPixelFormat': ['gray8', 'rgb8', 'bgr8', 'rgba8', 'bgra8'],
+    'Image.ColorSpaceGamma': ['linear', 'srgb'],
+    'Image.NominalPixelRange': ['nominalrange_0_255', 'normalized_0_1', 'normalized_1_1', 'nominalrange_16_235'],
+})
+
+
```

Comment:
I included the metadata_props.py file from the onnxmltools/utils as part of onnxtk since the dependency should be onnxtk -> onnxmltools, but not the other way around. _topology.py, in the common folder, depends on add_metadata_props.